x/group/README.md

```diff
@@ -0,0 +1,67 @@
+# Group Module
+
+## Group
+
+A group is simply an aggregation of accounts with associated weights. It is not
+an account and doesn't have a balance. It doesn't in and of itself have any
+sort of voting or decision power. It does have an "administrator" which has
+the power to add, remove and update members in the group. Note that a
+group account could be an administrator of a group.
+
+## Group Account
+
+A group account is an account associated with a group and a decision policy.
+Group accounts are abstracted from groups because a single group may have
+multiple decision policies for different types of actions. Managing group
+membership separately from decision policies results in the least overhead
+and keeps membership consistent across different policies. The pattern that
+is recommended is to have a single master group account for a given group,
+and then to create separate group accounts with different decision policies
+and delegate the desired permissions to from the master account to
+those "sub-accounts" using the `msg_authorization` module.
```

Comment:
How can we use the group account to `Authorize` users (other accounts) for some `messages`? Let's say I want to use the group account as the main account (granter) and grant some permissions to an account (grantee) to transfer funds. Who needs to sign this `authorization` transaction (using `x/authz`'s `grant` transaction)?

Reply:
the group account could be seen as a multisig account